Add a note on adding complex64/128 in a future version (#105)
data-apis/array-api: spec/API_specification/data_types.md

## float64

IEEE 754 double-precision (64-bit) binary floating-point number (see IEEE 754-2019).

:::{admonition} Future extension
:class: hint

> It is expected that in a future version of this standard, `complex64` and `complex128`

Reviewer: ```suggestion
It is expected that in the next version of this standard, `complex64` and `complex128`
```
